README.md

```diff
@@ -1,6 +1,6 @@
 # MCAP file format
 
-Working specification: https://docs.google.com/document/d/1NaC2v0Qlx43661XkrlVncybYiuloDFRtwdyDKlPpS-A/edit
+* [Draft specification](./docs/mcap)
```

Comment:
Naming wise, not sure there is any point in calling the folder `mcap`, since the repo is already called mcap.

Reply:
moved to docs/specification